tests/gen/Generated.hack

```diff
@@ -26,6 +27,20 @@ abstract final class Schema extends \Slack\GraphQL\BaseSchema {
 
     return await Dict\from_async($data);
   }
+
+  public static async function resolveMutation(
```

Comment:
I tried moving this to the base schema with a `resolveObject` method that both `resolveQuery` and `resolveMutation` called but got a type error since `self::resolveField` takes `self::THackType` as an argument.

Reply:
Might be possible with generics and `where` constraints, but I'm not sure.